internal/fourslash/test_parser.go

```diff
@@ -0,0 +1,258 @@
+package fourslash
+
+import (
+	"strings"
+	"testing"
+	"unicode/utf8"
+
+	"github.com/microsoft/typescript-go/internal/core"
+	"github.com/microsoft/typescript-go/internal/ls"
+	"github.com/microsoft/typescript-go/internal/lsp/lsproto"
+	runner "github.com/microsoft/typescript-go/internal/testrunner"
```

Comment:
We should rename the `testrunner` package internally, it shouldn't be called `runner`. I think I must have missed it when I moved it up a dir.